### ISSUE TITLE
toggleable setting to default on transmission enabled

### ISSUE THREAD

**Issue by Gordon-Dry** [steamp0rt/Kerbalism#59]
When adding science to the transmission queue I sometimes ask myself, why is it not just set to be transmitted by default?

So I request this as a setting in the settings.cfg:

TransmissionDefault = True

But ofc it would be just a little bit better if this behaviour could be toggled ingame.

**Comment by Gordon-Dry** [comment]
Just for further explanation; some "science management mods" like ScienceAlert pop up the message that a science is available again just after it has been transferred to the Kerbalism queue.
If it would be transmitted immediately (as long as a connection exists) this would not happen "so badly".

But I should mention that I also opened an issue on ScienceAlert because of this, see:
https://github.com/linuxgurugamer/ScienceAlert/issues/4
This one has already been closed, but IMHO it shouldn't been.
